client/src/main/java/tds/assessment/Assessment.java

```diff
@@ -51,7 +61,7 @@ public String getAssessmentId() {
      * @return {@code true} if the assessment is segmented
      */
     public boolean isSegmented() {
-        return segmented;
+        return segments.size() > 1;
```

Comment:
Actually, kept this as size > 1 since we decided to have a non-segmented (single segment) assessment contain its own segment.